src/python/WMCore/Wrappers/JsonWrapper/JSONThunker.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
                                  str,
                                  bytes,
                                  unicode,
```

Comment:
unicode is not defined in python3 therefore should be removed manually. At the same time bytes is not strings anymore. Therefore I would review this class manually.